plugins/intel-gsc/README.md

```diff
@@ -0,0 +1,37 @@
+# Intel Graphics System Controller
+
+## Introduction
+
+This plugin is used to update the Intel graphics system controller via the Intel Management Engine.
```

Comment:
There is another ME on the graphics card or you can only update graphics card firmware on a system with a host ME?

Reply:
> you can only update graphics card firmware on a system with a host ME

Correct, known issue, _possibly_ by design.

Reply:
I think it should be in the readme then

Reply:
Fixed.